.github/workflows/snow-cr-notification.js

```diff
@@ -0,0 +1,44 @@
+const { slackNotification, getLocalConfigs } = require('./helpers.js');
+
+async function main({ github, context, transaction_id, change_id, planned_start_time, planned_end_time } = {}) {
+    if (!github || !context) {
+        throw new Error("GitHub context is missing. Ensure you are running in the correct environment.");
+    }
+
+    if (process.env.LOCAL_RUN) {
+        console.log("Local run detected. Loading local configurations...");
+        const localConfigs = getLocalConfigs();
+        github = localConfigs.github;
+        context = localConfigs.context;
+    }
+
+    const { pull_request } = context.payload;
+    if (!pull_request) {
+        console.log("No pull_request found in context payload. Skipping notification.");
+        return;
+    }
+
+    const { number, title, html_url } = pull_request;
+    const isOpen = process.env.PR_STATE === 'open';
+    const prefix = isOpen
+        ? ':exclamation: ServiceNow Change Request Created and in progress: Transaction ID: ' + transaction_id + '\n: Planned Start Time: ' + planned_start_time + '\n: Planned End Time: ' + planned_end_time + '\n:'
+        : ':exclamation: ServiceNow Change Request Closed: Search for Change Record by Change ID: ' + change_id + ' or search for it by Planned Start Time and/or Planned End Time\n:';
```

Comment:
Should we keep the exclamation mark, or have more of an informational icon? the exclamation mark looks like something when wrong or needs immediate attention.

This is more informational that we requested a CMR 🤔

Reply:
good point, there is actually a servicenow icon, so I think I will go with that to easily find them amongst many other info icons.